_posts/2021-07-15-AI-fairness/AI_fairness_metrics.Rmd

```diff
@@ -0,0 +1,415 @@
+statistical independence. Following @barocas, these are:
+
+-   Separation: Given true target $Y$, prediction $\hat{Y}$ is
+    independent of group membership ($\hat{Y} \perp A | Y$).
+
+-   Sufficiency: Given prediction $\hat{Y}$, target $Y$ is independent
+    of group membership ($Y \perp A | \hat{Y}$).
+
+Given those two fairness criteria -- and two sets of corresponding
+metrics -- the natural question arises: Can we satisfy both? Above, I
+was mentioning precision and recall on purpose: to maybe "prime" you to
+think in the direction of "precision-recall trade-off". And really,
+these two categories reflect different preferences; usually, it is
+impossible to optimize for both. The most famous, probably, result is
+due to @2016arXiv161007524C : It says that predictive parity (testing
+for sufficiency) is incompatible with error rate balance (separation)
+when prevalence differs across groups. This is a theorem (yes, we're in
+the realm of theorems and proofs here) that may not be surprising, in
+light of Bayes' theorem, but is of great practical importance
+nonetheless: Unequal prevalence usually is the norm, not the exception.
+
+This necessarily means we have to make a choice. And this is where the
+theorems and proofs *do* matter. For example, @abs-1808-08619 show that
+in this framework -- the strictly technical approach to fairness --
+separation should be preferred over sufficiency, because the latter
+allows for arbitrary disparity amplification. Thus, *in this framework*,
+we may have to work through the theorems.
+
+What is the alternative?
+
+### Fairness, viewed as a social construct
+
+Starting with what I just wrote: No-one will likely challenge fairness
+*being* a social construct. But what does that entail?
+
+Let me start with a biographical reminiscence. In undergraduate
+psychology (a long time ago), probably the most-hammered-in distinction
+relevant to experiment planning was that between a hypothesis and its
+operationalization. The hypothesis is what you want to substantiate,
+conceptually; the operationalization is what you measure. There
+necessarily can't be a one-to-one correspondence; we're just striving to
+implement the best operationalization possible.
+
+In the world of datasets and algorithms, all we have are measurements.
+And often, these are treated *as though* they were the concepts. This
+will get more concrete with an example, and we'll stay with the hiring
+software scenario.
+
+Assume the dataset used for training, assembled from scoring previous
+employees, contains a set of predictors (among which, high school
+grades) and a target variable, say an indicator whether an employee did
+"survive" probation. There is a concept -- measurement mismatch on both
+sides.
+
+For one, say the grades are intended to reflect ability to learn, and
+motivation to learn. But depending on the circumstances, there sometimes
+are influence factors of much higher impact: socioeconomic status;
```

Comment:
I think you can just use commas as delimiters in this list, since none of them have compound clauses.

Reply:
right